packages/block-editor/src/components/line-height-control/style.scss

```
@@ -1,8 +1,21 @@
-.block-editor-line-height-control {
+// TODO: Remove this class before merging
+.block-editor-line-height-control-legacy {
 	margin-bottom: 24px;
 
 	input {
 		display: block;
 		max-width: 60px;
 	}
 }
+// end TODO
+
+.block-editor-line-height-control {
```

We should probably add a comment to signal that we should remove hardcoded classnames once we convert both this component and `InputControl` to use CSS-in-JS.

```suggestion
// TODO: remove hardcoded classnames
.block-editor-line-height-control {
```